zerver/openapi/zulip.yaml

```diff
@@ -6561,6 +6566,21 @@ paths:
           schema:
             type: integer
           example: 43
+        - name: check_time_limit_constraints
+          in: query
+          description: |
+            Whether to check that the messages requested to move within the stream
+            were older than 3 days limit for editing topics.
+
+            If this parameter is passed as `True`, server returns error if there are
+            messages that were older than 3 days. Otherwise, the server ignores the
+            messages older than 3 days and moves rest of the messages to the new topic.
+
+            **Changes**: New in Zulip 7.0 (feature level 160).
```

Comment:
I think the wording here and above can be improved. Would appreciate any suggestions to do so.